inventory/group_vars/curate/main.yml

```diff
@@ -51,6 +51,7 @@ dlp_curate_dotenv:
   solr_url: "{{ __dlp_curate_dotenv.solr_url | d ('') }}" 
   redis_host: localhost
   redis_port: 6379
+  honeybadger_api_key: "{{ vault_dlp.honeybadger_api_key | d ('') }}"
```

Comment:
My assumption here is that this will add `HONEYBADGER_API_KEY` to any curate instance, making it available for use in config yaml.

Reply:
Yes it will, after they've been rebuilt.